tests/sceneManager_test.cpp

```diff
@@ -11,6 +11,68 @@ using objects::Object;
 using objects::TriangleObj;
 
 const double kSkipFreq = 1000;
+const core::Vec3 kSkipVec3(0, 0, 0);
 generators::FakeRandomGen fakeRand;
```

Comment:
Remove?

Reply:
done, but I need to figure out how to make skipVec at one point in code, to make code more readable

Reply:
I still see fakeRand here. Have you actually removed it?